src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition.